site/content/how-to-monetize-apps/monetization.md

```diff
@@ -0,0 +1,92 @@
+---
+weight: 10 
+---
+
+# How to monetize apps?
+
+<!--Abstract, short intro -->
+Whether your app is already up and running or you're just starting off, the topic of **app monetization** will eventually cross your mind. If you don't know how to go about generating income from your apps, this is the place for you to start.
+
+
+<!-- Intro for newcomers -->
+## Introduction
+If you're new to LiveChat, here's how things work in here. LiveChat can be extended with various applications built by developers. All of this is done to meet the specific business needs of those who use LiveChat. You, as a developer and app author, can sell it on our [Marketplace](https://www.livechatinc.com/marketplace/). 
+
+If you'd like to know what types of applications can be built upon LiveChat Platform, check out the [Building LiveChat apps](/building-apps-for-livechat/) doc. 
```

Comment:
I'd consider putting the action first:
>Check out the Building LiveChat apps doc to learn what types of apps can be built upon LiveChat Platform.

I feel that _do [this] to [this]_ pattern is a bit more straightforward.

Reply:
This pattern is more intuitive, I agree. But here's the logic behind it:

"to" goes first in a sentence to explain what the result of an action is.
Then, if you decide that you want to achieve this result, you go to "do" (action, instructions). 

In _do [this] to [this]_ pattern, it may happen that someone wastes time reading what to do, only to discover that he/she doesn't want to achieve that result. 

But developers might be impatient and care more about action, so maybe it's a good idea to change it  :)

Reply:
Good point, you're totally right!  🙌 Let's stick with _To know... check out..._